dtest_setup.py

```diff
@@ -253,6 +253,9 @@ def patient_cql_connection(self, node, keyspace=None,
 
         return session
 
+    def get_cql_connection(self, node, **kwargs):
+        return self.patient_exclusive_cql_connection(node, retry_policy=None, **kwargs)
```

Comment:
this utility function was defined twice in two subclasses, so moved it up to the parent to make it reusable